package.json

```diff
@@ -38,6 +38,11 @@
           "type": "boolean",
           "default": true,
           "description": "Show code snippets for C++ when presenting possible GUID formats to insert."
+        },
+        "insertGuid.pasteAutomatically": {
+          "type": "string",
+          "default": "",
+          "description":"Paste GUID without selection, using the specified format. Available GUID formats are:\n\t{d}|{D}: Simple string in lowercase or uppercase\n\t{n}|{N}: Unformatted string in lowercase or uppercase\n\nOther characters will be interpreted literally.\nExamples:\n\t'{n}'\n\t\t=> '03afd966b18446079325cd58313ad637'\n\tnew GUID(\"{D}\")]\n\t\t => new GUID(\"123CF48A-16ED-469D-B38E-A465DDCD667E\")]\n"
```

Comment:
IIRC, while I like the documented values, you can also list the possible values here such that Code will render them in the settings menu. That would be ideal, but not a blocker if not included.

Reply:
Per [the docs](https://code.visualstudio.com/api/references/contribution-points#contributes.configuration), it looks like there is only something for a dropdown enum (though if you know otherwise just point me in the right direction, this is my first foray into vscode extension development :)) . I was able to get good results by switching to markdown though:

![image](https://user-images.githubusercontent.com/1996525/122184400-246eff80-cec7-11eb-8681-87c9df096f9d.png)